spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -1233,7 +1258,7 @@ protected void executeListener(Channel channel, Message messageIn) throws Except
 					if (length < 0 || length > byteBuffer.remaining()) {
 						throw new ListenerExecutionFailedException("Bad batched message received",
 								new MessageConversionException("Insufficient batch data at offset " + byteBuffer.position()),
-										message);
+								message);
 					}
```

Comment:
Why? That was bad indention before. The `message` arg is fully identical for the `new MessageConversionException` .
Therefore, that is logical that they should be aligned